miio/airhumidifier_jsqs.py

```diff
@@ -0,0 +1,249 @@
+        {'did': 'water_shortage_fault', 'siid': 7, 'piid': 1, 'code': 0, 'value': False},
+        {'did': 'tank_filed', 'siid': 7, 'piid': 2, 'code': 0, 'value': False},
+        {'did': 'overwet_protect', 'siid': 7, 'piid': 3, 'code': 0, 'value': False}
+    ]
+    """
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
+
+    # Air Humidifier
+
+    @property
+    def is_on(self) -> bool:
+        """Return True if device is on."""
+        return self.data["power"]
+
+    @property
+    def power(self) -> str:
+        """Return power state."""
+        return "on" if self.is_on else "off"
+
+    @property
+    def error(self) -> int:
+        """Return error state."""
+        return self.data["fault"]
+
+    @property
+    def mode(self) -> OperationMode:
+        """Return current operation mode."""
+
+        try:
+            mode = OperationMode(self.data["mode"])
+        except ValueError as e:
+            _LOGGER.exception("Cannot parse mode: %s", e)
+            return OperationMode.Auto
+
+        return mode
+
+    @property
+    def target_humidity(self) -> int:
+        """Return target humidity."""
+        return self.data["target_humidity"]
+
+    # Environment
+
+    @property
+    def relative_humidity(self) -> int:
+        """Return current humidity."""
+        return self.data["relative_humidity"]
+
+    @property
+    def temperature(self) -> Optional[float]:
+        """Return current temperature, if available."""
+        if self.data["temperature"] is not None:
+            return round(self.data["temperature"], 1)
+        return None
+
+    # Alarm
+
+    @property
+    def buzzer(self) -> Optional[bool]:
+        """Return True if buzzer is on."""
+        if self.data["buzzer"] is not None:
+            return self.data["buzzer"]
+        return None
+
+    # Indicator Light
+
+    @property
+    def led_light(self) -> Optional[bool]:
+        """Return status of the LED."""
+
+        if self.data["led_light"] is not None:
+            return self.data["led_light"]
+        return None
+
+    # Other
+
+    @property
+    def tank_filed(self) -> Optional[bool]:
+        """Return the tank filed."""
+
+        if self.data["tank_filed"] is not None:
+            return self.data["tank_filed"]
+        return None
+
+    @property
+    def water_shortage_fault(self) -> Optional[bool]:
+        """Return water shortage fault."""
+
+        if self.data["water_shortage_fault"] is not None:
+            return self.data["water_shortage_fault"]
+        return None
+
+    @property
+    def overwet_protect(self) -> Optional[bool]:
+        """Return True if overwet mode is active."""
+
+        if self.data["overwet_protect"] is not None:
+            return self.data["overwet_protect"]
+        return None
+
+
+class AirHumidifierJsqs(MiotDevice):
+    """Main class representing the air humidifier which uses MIoT protocol."""
+
+    mapping = _MAPPING
```

Comment:
Please add `_supported_models` here, it should be a list of model strings supported by this integration.